src/anemoi/training/diagnostics/callbacks/plot.py

```diff
@@ -938,7 +938,7 @@ def _plot(
             torch.cat(tuple(x[self.sample_idx : self.sample_idx + 1, ...].cpu() for x in outputs[1])),
             in_place=False,
         )
-        output_tensor = pl_module.output_mask.apply(output_tensor, dim=2, fill_value=np.nan).numpy()
+        output_tensor = pl_module.output_mask.apply(output_tensor, dim=1, fill_value=np.nan).numpy()
```

Comment:
Why does this dim change?

Reply:
It didn't work (LAM + `plot: detailed`). It seems that the `output_tensor` changed its shape at some point.